src/stream/src/executor/top_n/top_n_cache.rs

```diff
@@ -214,6 +273,20 @@ impl TopNCacheTrait for TopNCache<false> {
         };
 
         if !self.is_high_cache_full() {
+            let need_refill = match self.high.last_key_value() {
+                Some(high_last) => elem_to_compare_with_high.0 > *high_last.0,
+                None => true,
+            };
+            if need_refill {
+                managed_state
+                    .fill_high_cache(
+                        group_key,
+                        self,
+                        self.middle.last_key_value().unwrap().0.clone(),
+                        self.high_capacity,
+                    )
+                    .await?;
+            }
```

Comment:
it seems that we fill the high cache too many times, which makes the e2e test timeout :thinking: